cli/auth/auth_command.py

```diff
@@ -1,14 +1,23 @@
+import json
+
 import click
 import traceback
+
 from cli.auth.auth_manager import AuthManager
-from cli.exceptions.custom_exceptions import AuthProcessError, CycodeError
+from cli.user_settings.credentials_manager import CredentialsManager
+from cli.exceptions.custom_exceptions import AuthProcessError, NetworkError, HttpUnauthorizedError
 from cyclient import logger
+from cyclient.cycode_token_based_client import CycodeTokenBasedClient
 
 
-@click.command()
+@click.group(invoke_without_command=True)
```

Comment:
maybe there is a better way instead of set the command as a group?

Reply:
I did it to save backward comparability. It should look like our scan commands:
- scan path ...
- scan repository ...

the best-looking case:
- auth login
- auth check

but due to the already existing command `auth` (not the group of commands), we should do `invoke_without_command` trick to save prev usage of the command.

the other way to workaround it is to move the check command out of the "auth" space

for example: `cycode status` looks also great! and we can return many things on it (same with git status)! wdyt? @MichalBor

Reply:
@artem-fedorov cc

Reply:
@MarshalX thanks for the great explanation! 
1. lets continue with `cycode auth` and `cycode auth check` as planned, seems that the decorator you suggested is the best option at the moment :)
2. regarding `cycode status` - good idea! lets add it to the roadmap

Reply:
you are welcome :)

1. is pr approved?) 
2. added